boards/st/sensortile_box_pro/Kconfig

```
@@ -0,0 +1,12 @@
+# SENSORTILE_BOX_PRO board configuration
+
+# Copyright (c) 2024 STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_SENSORTILE_BOX_PRO
+
+config BOARD_SERIAL_BACKEND_CDC_ACM
+	bool "USB CDC"
```

Nit: this could be a bit more verbose.
```suggestion
	bool "Use USB CDC as serial console backend"
```